build/single.html

```diff
@@ -6,21 +6,63 @@
     <link rel="stylesheet" type="text/css" href="style.css" />
     <link rel="stylesheet" type="text/css" href="styles/responsive.css" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,700" rel="stylesheet" />
+    <link rel="stylesheet" href="http://i.icomoon.io/public/48159919ac/test/style.css"></link>
```

Comment:
Remove this test icomoon link. The main issue you're running into is you have added the fonts directory to the root directory and it should be inside the `build` directory.

https://github.com/mattcsloan/info-7sig/pull/2
I have just opened a pull request that has icomoon working. You can go ahead and remove all your icomoon changes and pull those updates into your branch by doing the following:

1. Commit any changes you've made since your last commit so you don't lose any work. ```git commit -am"Updates to banner, and page columns"```
2. Delete your fonts directory and then ```git commit -am"removed fonts directory"```
3. Fetch the latest branches with ```git fetch```
4. Add the latest code from my icomoon-additions branch into your branch: ```git pull origin icomoon-additions```
5. If you have any merge conflicts, it will tell you to resolve them. The only file that could possibly have a conflict is the single.html file. If you need help resolving conflicts, just let me know. Once you resolve conflicts (if there were any), you'll likely need to do a ```git add build/single.html``` then ```git commit -am"merge from icomoon-additions"``` then ```git push origin template```

Reply:
Can you explain the difference between fit fetch and git pull.  I'm reading the documentation and it's telling me git pull is a fetch and a merge.  I am curious as to why I should do a fetch and then pull origin icomoon-additions?